### PR TITLE
Feature: Add Dutch localization

### PR DESCRIPTION
**Pull request by Lemii**
This PR adds Dutch localization to the Lisk Desktop application.

## How have I implemented/fixed it?

The English `common.json` was used as a template.

Small side note: it was required to use the `nl-NL` locale notation. When using `nl`, the app would crash after signing in. This is most like an 'issue' with `numeral`.

### How has this been tested?

This feature branch has been tested in a local dev environment.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-desktop/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-desktop/blob/development/docs/CSS_GUIDE.md)

### CODE REVIEW

File: i18n/locales/nl-NL/common.json [hunk 1]
```diff
@@ -0,0 +1,565 @@
+{
+  " - Confirm transaction on your {{deviceModel}}": " - Bevestig de transaction op je {{deviceModel}}",
+  " LSK": " LSK",
+  " Make sure that you are using the latest version of Lisk.": " Controleer of je de laatste versie van Lisk gebruikt.",
+  " is out. ": " is uit. ",
+  "% of successfully forged blocks in relation to total blocks that were available for this particular delegate to forge": "% van succesvol gesmede blokken in relatie tot het totaal aantal blokken die beschikbaar waren om te smeden voor de afgevaardigde",
```
Comment (michielmulders) [inline i18n/locales/nl-NL/common.json]: `gesmede` sounds a bit much like `smithing` 🗡 
Maybe better to use here the original `forged` as it's like a term associated with Lisk.

File: i18n/locales/nl-NL/common.json [hunk 2]
```diff
@@ -0,0 +1,565 @@
...
+  "+ Transaction fee {{fee}} LSK": "+ Transactie fee {{fee}} LSK",
+  "2nd passphrase registration": "2e wachtwoordzin registratie",
+  "2nd passphrase registration failed": "2e wachtwoordzin registratie is mislukt",
```
Comment (michielmulders) [inline i18n/locales/nl-NL/common.json]: Getting creative with `wachtwoordzin`? :)

**Comment by Lemii** [comment]
Thanks for the review @michielmulders . You can expect an update to the PR soon 👍

**Comment by reyraa** [comment]
Hi there, as we have already communicated on Discord about not implementing the language support, we’ll close this PR.
For those that are unaware about this, our current strategy dictates to only implement one language which is English. This decision is based on several reasons that have been explained.
We want to thank you for showing support and taking the effort to open up this PR but unfortunately we can’t merge your PR at this point.

Thanks for understanding,
the LiskHQ team

**Pull request closed without merging**